docs/source/serialization.rst

```diff
@@ -193,3 +193,56 @@ Specify relations by intid:
     "relatedItems": [347127075, 347127076]
   }
 
+Walking the content
+-------------------
+
+The serialized content have some hints on how to move to the items around.
```

Comment:
"The response body of a GET request contains three attributes that allows navigating to the parent and the next and previous sibling in the container the current document is located."